src/Schema/Subscription.js

```diff
@@ -28,9 +28,11 @@ const Subscription = new Schema({
     firstBillingDate: Date,
     nextBillingDate: Date,
     paidThroughDate: Date,
-    status: String,
+    status: {
```

Comment:
Do we need this? I think it would be easier to do it like
```javascript
status: {
    type: String,
    enum: SubscriptionStatus.statuses,
}
```
And remain consistent with both subscriptions-braintree and braintree itself